config.js

```diff
@@ -7,6 +7,12 @@ config.server = {
     lifetimeCheckInterval: 1000,        // Minimum interval between lifetime checks in ms
```

Comment:
An entry in CHANGES_NEXT_RELEASE describing the change introduced by this PR should be added.

Reply:
Fixed by b934729

(It is a good practice from traceability reasons to include a mark like this specifying that the comment has been addressed and can be considered closed. I'm doint it for you this time :), but it would be great if you could do in the future for other PRs. Thanks!)

Reply:
I fully agree. It will be nice to write it down in the contributions guidelines. Thank you :)

Reply:
The "pull request protocol" is documented, but I'm afraid not in this repo but in this other: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol

A migration from there to the contributions guide in this repository is welcome! Basically it's a matter of copy-pasting and change repository name literals :)